infrastructure/perf/artillery.yml

```diff
@@ -0,0 +1,14 @@
+config:
+  # target: 'http://localhost:3000'
+  target: "https://staging.khouryofficehours.com"
+  phases:
+    - duration: 60
+      arrivalRate: 30
+scenarios:
+  - flow:
```

Comment:
The other endpoint that might be worth testing is a POST to the `/api/v1/khoury_login` since it does a lot of db queries